### PR TITLE
doc: configure bin-packing in helm chart

### LINKED ISSUE #448: Configure bin-packing in Helm chart
Hello,

Any pointers on how to enable packing? I read kubernetes has [bin-packing](https://kubernetes.io/docs/concepts/scheduling-eviction/resource-bin-packing/#enabling-bin-packing-using-requestedtocapacityratio), how do we use such plugins in co-scheduler using helm commands?

### PR DESCRIPTION
**Pull request by Gekko0114**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Added the explanation on how to configure bin-packing in helm chart.

#### Which issue(s) this PR fixes:
Fixes #448 

#### Special notes for your reviewer:
Thank you for reviewing this PR.
Please let me know if I have made any mistakes or if there are any improvements that could be made.
Your comments and feedback are appreciated.

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 1]
```diff
@@ -54,3 +55,62 @@ The following table lists the configurable parameters of the as-a-second-schedul
 | `controller.replicaCount` | Controller replicaCount     | `1`                                                                                             |
 | `plugins.enabled`         | Plugins enabled by default  | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
 | `plugins.disabled`        | Plugins disabled by default | `["PrioritySort"]`                                                                              |
+
+###  Configure bin-packing in Helm chart
+If you want to use bin-packing in the helm chart, you need to add code similar to the following in the `pluginConfig` section of `values.yaml`.
+```yaml
```
Comment (zwpaper) [inline manifests/install/charts/as-a-second-scheduler/README.md:61]: please add an empty line between the doc and the code

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 2]
```diff
@@ -54,3 +55,62 @@ The following table lists the configurable parameters of the as-a-second-schedul
 | `controller.replicaCount` | Controller replicaCount     | `1`                                                                                             |
 | `plugins.enabled`         | Plugins enabled by default  | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
 | `plugins.disabled`        | Plugins disabled by default | `["PrioritySort"]`                                                                              |
+
+###  Configure bin-packing in Helm chart
+If you want to use bin-packing in the helm chart, you need to add code similar to the following in the `pluginConfig` section of `values.yaml`.
```
Comment (zwpaper) [inline manifests/install/charts/as-a-second-scheduler/README.md:60]:
```suggestion
If you want to enable [bin-packing](https://kubernetes.io/docs/concepts/scheduling-eviction/resource-bin-packing/#enabling-bin-packing-using-requestedtocapacityratio) in the helm chart, you need to configure it similar to the following in `pluginConfig` section of `values.yaml`.
```

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 3]
```diff
@@ -54,3 +55,62 @@ The following table lists the configurable parameters of the as-a-second-schedul
...
+      resources:
+      - name: intel.com/foo
+        weight: 3
+      - name: intel.com/bar
+        weight: 3
+      type: RequestedToCapacityRatio
+```
+
+Then the ConfigMap will be created.
+```yaml
```
Comment (zwpaper) [inline manifests/install/charts/as-a-second-scheduler/README.md:84]: need an empty line

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 4]
```diff
@@ -54,3 +55,62 @@ The following table lists the configurable parameters of the as-a-second-schedul
...
+          score: 10
+      resources:
+      - name: intel.com/foo
+        weight: 3
+      - name: intel.com/bar
+        weight: 3
+      type: RequestedToCapacityRatio
+```
+
+Then the ConfigMap will be created.
```
Comment (zwpaper) [inline manifests/install/charts/as-a-second-scheduler/README.md:83]:
```suggestion
Then the ConfigMap containing `KubeSchedulerConfiguration` will be created:
```

**Review by zwpaper — CHANGES_REQUESTED** [review]
one more question, did you test the functionality locally?

**Comment by Huang-Wei** [comment]
@Gekko0114 Apologize that I should have commented in the original issue that it's not planned to support it in scheduler-plugins. (and hence remote the /help label)

The reason is that bin-packing is supported via one of the in-tree plugins, so the problem domain applies to the upstream default-scheduler, instead of any scheduler plugin that adopts it.

/close not-planned

**Comment by zwpaper** [comment]
closing due to not planned, check https://github.com/kubernetes-sigs/scheduler-plugins/issues/448

/close

**Comment by Gekko0114** [comment]
Got it! Thank you for the information

**Pull request closed without merging**